generator.go

```diff
@@ -272,92 +260,59 @@ func (g *Gen) NewV4() (UUID, error) {
 func (g *Gen) NewV5(ns UUID, name string) UUID {
 	u := newFromHash(sha1.New(), ns, name)
 	u.SetVersion(V5)
-	u.SetVariant(VariantRFC4122)
+	u.SetVariant(VariantRFC9562)
 
 	return u
 }
 
 // NewV6 returns a k-sortable UUID based on a timestamp and 48 bits of
 // pseudorandom data. The timestamp in a V6 UUID is the same as V1, with the bit
 // order being adjusted to allow the UUID to be k-sortable.
-//
-// This is implemented based on revision 03 of the Peabody UUID draft, and may
-// be subject to change pending further revisions. Until the final specification
-// revision is finished, changes required to implement updates to the spec will
-// not be considered a breaking change. They will happen as a minor version
-// releases until the spec is final.
 func (g *Gen) NewV6() (UUID, error) {
+	/* https://datatracker.ietf.org/doc/html/rfc9562#name-uuid-version-6
```

Comment:
Nit: maybe include the source link in the godoc comment so that someone doesn't have to dive into the code to find it.

Reply:
@dylan-bourque I thought about that, but I think it's too much information that most users won't care about. The link is only there to show the origin of the bit layout below it, which is not something that needs to be in the godoc. The README has links to the full RFC, and that shows up in the godoc already.